MediaBrowser.Api/ItemUpdateService.cs

```diff
@@ -198,6 +198,12 @@ private List<NameValuePair> GetContentTypeOptions(bool isForItem)
 
         public void Post(UpdateItem request)
         {
+            if (request.ItemId == "*")
+            {
+                // Special case: Refresh everything in database. Probably not a great idea to run often.
+                _libraryManager.UpdateAll();
+                return;
+            }
```

Comment:
I have no idea how I could make Jellyfin automatically run this after it updates to a version with blurhash support. I suppose it would be a migration, but I'm not familiar with how those work in this codebase yet. Any tips?

Reply:
@GranPC There is support in the server for adding a code migration that will be guaranteed to run a single time. There are two existing migrations already you can take a look at: https://github.com/jellyfin/jellyfin/tree/master/Jellyfin.Server/Migrations/Routines

Reply:
I don't see why we need a migration. Users should rescan their library manually if they wish, or wait for a nightly task.